README.md

```diff
@@ -317,6 +318,25 @@ Component structure:
 
 [↑ back to Table of Contents](#table-of-contents)
 
+## Component event names
+
+Vue.js provides events scoped to the component it emits and they are named as the developers desires. `~Make this paragraph better`
+
+### Why?
+
+* Developers are free to use native likes event names and it can cause confusion down the line;
+* The freedom of naming events can lead to a DOM templates incompatibility;
```

Comment:
I would add a link to the [DOM Template Parsing Caveats](https://vuejs.org/v2/guide/components.html#DOM-Template-Parsing-Caveats)